### DISCUSSION TITLE
documentation: how to use Health Check

### DISCUSSION THREAD

**Discussion by knwpsk** [zwave-js/zwave-js#4939] [General]
It would be great if we could enhance the documentation and give end users a guide on how to use the Health Check function in zWaveJS.

https://github.com/zwave-js/node-zwave-js/blob/master/docs/troubleshooting/network-health.md
This page suggests that it's useful, but gives the user (like me) no idea how to find it or use it.

(I hesitate to start editing the page myself, being new here, and new to zwave also.)

In my troubleshooting, [AlCalzone](https://github.com/AlCalzone) was very kind to guide me through it.

**Comment by AlCalzone** [comment]
Related: https://github.com/zwave-js/zwavejs2mqtt/issues/2598
I think the UI/usage side of things should be documented there, but we can link to it from here.
